templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -881,7 +877,7 @@ Resources:
                   - ATL_ENABLED_PRODUCTS=
                   - ATL_ENABLED_SHARED_HOMES=Bitbucket
                   - ATL_APP_NFS_SERVER=true
-                  - ATL_NFS_SERVER_DEVICE=/dev/sdf
+                  - ATL_NFS_SERVER_DEVICE=/dev/xvdf
```

Comment:
Is this somehow affected by the instance type and type of storage as well? E.g. NVMe storage blocks have different mapping as well: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html

Reply:
What's the significance of the device name? I did a bit of googling and don't really understand how all of this relates to the deployment of the volume 😕

Reply:
I don't think affects it much. IME the NVME volumes are separate from EBS, and not even used as root on the machines. I believe /dev/xvdX volumes are virtual, which is what EBS is (a SAN basically).

Reply:
The /dev/... device names usually pertain to the storage protocol, but with udev/systemd you can map them to whatever you want. On my desktop machine, I have 2 m.2 drive (linked directly into the PCIE bus), but one appears as /dev/nvme0, because it's a true NVM Express drive, but the other appears as /dev/sda, as it speaks SATA.

In AWS, the mapping can be overridden by the AMI or the CFN template. There's some stuff on it here: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/block-device-mapping-concepts.html